src/server/enterprise/cmds/cmds.go

```diff
@@ -1,359 +0,0 @@
-// Package cmds implements pachctl commands for the enterprise service.
-package cmds
-
-import (
-	"fmt"
-
-	"github.com/pachyderm/pachyderm/v2/src/admin"
-	"github.com/pachyderm/pachyderm/v2/src/enterprise"
-	"github.com/pachyderm/pachyderm/v2/src/internal/cmdutil"
-	"github.com/pachyderm/pachyderm/v2/src/internal/config"
-	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
-	"github.com/pachyderm/pachyderm/v2/src/internal/pachctl"
-	"github.com/pachyderm/pachyderm/v2/src/license"
-	"github.com/pachyderm/pachyderm/v2/src/version"
-	"github.com/spf13/cobra"
-)
-
-func getIsActiveContextEnterpriseServer() (bool, error) {
-	cfg, err := config.Read(false, true)
-	if err != nil {
-		return false, errors.Wrapf(err, "could not read config")
-	}
-	_, ctx, err := cfg.ActiveEnterpriseContext(true)
-	if err != nil {
-		return false, errors.Wrapf(err, "could not retrieve the enterprise context from the config")
-	}
-	return ctx.EnterpriseServer, nil
-}
-
-// DeactivateCmd returns a cobra.Command to deactivate the enterprise service.
-func DeactivateCmd(pachctlCfg *pachctl.Config) *cobra.Command {
```

Comment:
IMHO we should just return an error for each of these commands instead of delete them entirely, so that users aren't confused when this command isn't discoverable. cc @robert-uhl @FahadBSyed